### PR TITLE
Cmlh5 to netcdf

### PR DESCRIPTION
**Pull request by maxmargraf**
Add a function to read a cmlh5 file and parse data from each cml_id to a xarray dataset and return them as a list

### CODE REVIEW

**Comment by cchwala** [comment]
@maxmargraf Can you also add a tiny test?

**Comment by maxmargraf** [comment]
@cchwala should this test rely on the 'old' example cmlh5 file which is available in v0.2.5?

**Comment by cchwala** [comment]
Yes. Just read from the old example file, which we will keep anyway, and validate some data and metadata.

File: pycomlink/tests/test_cmlh5_to_netcdf.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import unittest
+from pycomlink.io.cmlh5 import read_from_cmlh5
```
Comment (cchwala) [inline pycomlink/tests/test_cmlh5_to_netcdf.py:2]: not required anymore

File: pycomlink/tests/test_cmlh5_to_netcdf.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+testdata_fn = str(get_test_data_path() + "/75_cmls_processed.h5")
+
+
+class Testcmlh5tonetcdf(unittest.TestCase):
+    def test_cmlh5_to_netcdf_fun(self):
+
+        # open cmlh5
+        cmlh5_file = read_from_cmlh5(testdata_fn)
```
Comment (cchwala) [inline pycomlink/tests/test_cmlh5_to_netcdf.py:19]: not required anymore

File: pycomlink/tests/test_cmlh5_to_netcdf.py (hunk 1 above)
Comment (cchwala) [reply pycomlink/tests/test_cmlh5_to_netcdf.py:2]: important, because this function uses Comlink objects which will not work in the future

Comment (maxmargraf) [reply pycomlink/tests/test_cmlh5_to_netcdf.py:2]: My mistake, it is not used in the testing, therefore will be delete in the next commit

File: pycomlink/io/cmlh5_to_netcdf.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    "rsl": ("time", cml_ch_g["rx"][:]),
+                },
+                coords={
+                    "time": cml_ch_g["time"][:],
+                    "channel_id": channel_name,
+                    "cml_id": cml_g.attrs["cml_id"],
+                    "site_a_latitude": cml_g.attrs["site_a_latitude"],
+                    "site_b_latitude": cml_g.attrs["site_b_latitude"],
+                    "site_a_longitude": cml_g.attrs["site_a_longitude"],
+                    "site_b_longitude": cml_g.attrs["site_b_latitude"],
```
Comment (cchwala) [inline pycomlink/io/cmlh5_to_netcdf.py:48]: Boing!

Fix will be part of #70

**Pull request merged**